cli/orchestration/tag-filter.md

```diff
@@ -11,30 +11,29 @@ The **Tag Filter** can be used in multiple Terramate features:
 - [stack.before](../stacks/configuration.md#stackbefore-setstringoptional)
 - `terramate <cmd> --tags <filter>`
 
-The filter returns a list of stacks containing `tags` that satisfy the filter
-query. The query language is best explained with some examples but a formal
+Using the `tags` filter, you can list or run commands on stacks based on tag conditions. Use `--tags` to filter stacks having specific tags and `--no-tags` to filter those without the specified tags. Both options support combining tag filters using AND (`:`) and OR (`,`) for more complex queries.
```

Comment:
```suggestion
Using the `tags` filter, you can list or run commands on stacks based on tag conditions. Use `--tags` to filter stacks having specific tags and `--no-tags` to filter those without the specified tags. The `--tags` option support combining filters using AND (`:`) and OR (`,`) for more complex queries, whereas `--no-tags` should be used to specify the list of tags that must not be present in the stacks list and then only `,` is supported.
```

Reply:
@i4ki Thanks for the correction! 😄  I'm closing this PR as the other PR (#93) makes the necessary changes.